ui/client/components/graph/node-modal/editors/expression/Duration/DurationEditor.tsx

```diff
@@ -13,35 +13,31 @@ export type Duration = {
   minutes: number,
 }
 
-export type DurationComponentType = {
-  label: string,
-  fieldName: string,
-}
-
 type Props = {
   expressionObj: ExpressionObj,
   onValueChange: Function,
   validators: Array<Validator>,
   showValidation?: boolean,
   readOnly: boolean,
   isMarked: boolean,
+  components: Array<string>,
 }
 
 const SPEL_DURATION_DECODE_REGEX = /^T\(java\.time\.Duration\)\.parse\('(.*?)'\)$/
 const SPEL_DURATION_SWITCHABLE_TO_REGEX = /^T\(java\.time\.Duration\)\.parse\('P([0-9]{1,}D)?(T([0-9]{1,}H)?([0-9]{1,}M)?)?'\)$/
 const SPEL_FORMATTED_DURATION = (isoDuration) => `T(java.time.Duration).parse('${isoDuration}')`
-const UNDEFINED_DURATION = {
-  days: () => undefined,
-  hours: () => undefined,
-  minutes: () => undefined
+const NULL_DURATION = {
```

Comment:
NONE_DURATION? I assume there can be null / undefined?

Reply:
Done